Demo/Sources/CastPlayerView.swift

```diff
@@ -173,16 +173,16 @@ private struct MainView: View {
 
 private struct MediaQueueView: View {
     @ObservedObject var mediaQueue: MediaQueue
-    @State private var currentItem: CastPlayerItem?
+    @State private var currentItemId: CastPlayerItem.ID?
 
     var body: some View {
-        List(mediaQueue.items, id: \.self, selection: $currentItem) { item in
+        List(mediaQueue.items, selection: $currentItemId) { item in
```

Comment:
Maybe renaming `currentItemId` as `selection` would be even more expressive here. What do you think?